.env.example

```diff
@@ -1,5 +1,3 @@
-PUBLIC_KEY=
```

Comment:
I think we should still offer the possibility to use public and secret keys. We need to document in case of restoring wallet using mnemonics this is not needed